src/App.tsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+// export function
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const { title, description } = movie;
```

Comment:
```suggestion
  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
```
Good, but you can make destructuring in arguments.

Reply:
agree